src/main/java/com/google/alpollo/SentimentServlet.java

```diff
@@ -0,0 +1,31 @@
+package com.google.alpollo;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.alpollo.model.SongSentiment;
+import com.google.cloud.language.v1.Sentiment;
+import com.google.gson.Gson;
+
+/**
+ * Sending a POST request with the lyrics of the song as a parameter will give the Client
+ * the overall positivity of the song and how strong the sentiment of the song is.
+ */
+@WebServlet("/sentiment")
+public class SentimentServlet extends HttpServlet {
+  private final Gson gson = new Gson();
+  private static String LYRICS;
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
The servlet shouldn't throw an `IOException`. Use a `try/catch` instead and in the `catch` block you send an error in the response

Reply:
Olya did this already in her separate branch, so when we merge it should be ok. Should I do it too though?

Reply:
In that case I'm OK with leaving this change for the next PR.